src/components/SearchableSingleSelect/SearchableSingleSelect.jsx

```diff
@@ -402,9 +415,22 @@ const SearchableSingleSelect = createClass({
 								? this.renderOptionContent(selectedOptionProps, '')
 								: selectedOptionProps.Selected
 						}
+						className={cx('&', {
+							'&-select-error': errorChildProps && errorChildProps.children && errorChildProps.children !== true
```

Comment:
I think this is the same logic issue as above that you fixed elsewhere. Also, should it be `&-select-is-error` instead?

Reply:
@jondlm my most recent changes should have this updated